### ISSUE TITLE
Vs code extension games 

### ISSUE THREAD

**Issue by sharafabacery** [OnlyBens/vscode-extension-ideas#5]
Making most famous games on web or Android and recreate using typescript and using github outh to track records (flappy bird or tic tac toe (can be multiplier) or T-Rex run (game in chrome)

**Comment by charfweh** [comment]
aw damn, I'd like mini games in vscode.

**Comment by sharafabacery** [comment]
Sorry for bad English

**Comment by charfweh** [comment]
> Sorry for bad English

it's okay, you made your point, its understandable :) cheers mate

**Comment by sudeepgumaste** [comment]
I would love a ping pong multiplayer where we can send game request using github id

**Comment by charfweh** [comment]
> I would love a ping pong multiplayer where we can send game request using github id

that's a good one, or hell even uno cards during scrum meetings

**Comment by sharafabacery** [comment]
Awesome guys❤️😂😂😂

**Comment by talmax1124** [comment]
@sudeepgumaste @charfweh @sharafabacery @benawad  We should all work on a game together

**Comment by dayvsonsales** [comment]
like MSN Messenger games. Checkers, chess, solitaire spider...

**Comment by sharafabacery** [comment]
My little concept, if there bug in code, take a rest and play mini game in same editor, like icon in left of editor when you click, will open window and play, like hellow world panel that Ben do in how to make vs code extension tut.

**Comment by charfweh** [comment]
@talmax1124 yeah surely, it'll be open source mate, so no worries. Also @sharafabacery yes exactly, mini games to take a little rest

**Comment by sharafabacery** [comment]
no problem in make this little extension, but it was my first time coding typescript when ben makes the extension, and I will my best if that happen. @dayvsonsales @sudeepgumaste @charfweh @talmax1124

**Comment by sharafabacery** [comment]
my experience backend development with node js,and some php.

**Comment by dayvsonsales** [comment]
> no problem in make this little extension, but it was my first time coding typescript when ben makes the extension, and I will my best if that happen. @dayvsonsales @sudeepgumaste @charfweh @talmax1124

have fun.
